content/en/_index.md

```diff
@@ -245,11 +347,20 @@ If the code snippet is platform specific, explicitly show how to use that
 platform's credentials.
  
 {{< tabpane langEqualsHeader=true >}}
- {{< tab header="Java" >}}
+  {{< tab header="Java" >}}
    // Most clients use ADC by default. However, if your application needs to showcase a specific
    // credential source, show users how to do that explicitly.
    GoogleCredentials credentials = GoogleCredentials.fromStream(new FileInputStream("/path/to/credentials.json"));
-{{< /tab >}}
+  {{< /tab >}}
+  {{< tab header="Node.js" >}}
+    // Most clients use ADC by default. However, if your application needs to showcase a specific
```

Comment:
Are there still official clients that don't use ADC by default?

Reply:
FWIW, the Java healthcare samples don't: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/306387ec973a6e97f733d602bd3c31c5dbab8cd7/healthcare/v1/src/main/java/snippets/healthcare/fhir/FhirStoreDelete.java#L57